src/main/java/proj/kedabra/billsnap/presentation/controllers/AccountController.java

```diff
@@ -96,4 +98,24 @@ public AccountResource getAccount(@AuthenticationPrincipal final Principal princ
         return mapper.toResource(accountFacade.getAccount(principal.getName()));
     }
 
+    @Cacheable(value = CacheNames.PROFILE, key = "#principal.name")
```

Comment:
Rather than cacheable, should @CachePut since the cacheable is on our get request